core/templates/dev/head/domain/learner_dashboard/learner_dashboard_icons_directive.html

```diff
@@ -1,4 +1,5 @@
 <i class="oppia-learner-dashboard-icon fa fa-clock-o"
+ng-if="shouldAddToPlaylistIcon()"
```

Comment:
indent to match left edge of "class" in previous line

Reply:
ok